<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu_ui</class>
 <widget class="QWidget" name="Menu_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="Menu">
      <widget class="QLabel" name="hwMenu_newworld">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>422</y>
         <width>350</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_newworld.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_quitsel">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>640</y>
         <width>300</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Images/wr/Images/wr/wr_menu_quitsel.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_loadworldsel">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>485</y>
         <width>350</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_loadworldsel.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_optionsmousearea">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>580</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_loadworldmousearea">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>510</y>
         <width>300</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_newworldmousearea">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>438</y>
         <width>301</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_quit">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>640</y>
         <width>300</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/Images/wr/Images/wr/wr_menu_quit.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_newworldsel">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>422</y>
         <width>350</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_newworldsel.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hw_menu_background">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1599</width>
         <height>900</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_background.jpg</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_optionssel">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>568</y>
         <width>350</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_optionsel.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_options">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>568</y>
         <width>350</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_options.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_quitmousearea">
       <property name="geometry">
        <rect>
         <x>728</x>
         <y>651</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="hwMenu_loadworld">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>485</y>
         <width>350</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_loadworld.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="hw_versionnumber">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>870</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: orange;</string>
       </property>
       <property name="text">
        <string>v. 0.1.1 alpha</string>
       </property>
      </widget>
      <zorder>hw_menu_background</zorder>
      <zorder>hwMenu_newworld</zorder>
      <zorder>hwMenu_quit</zorder>
      <zorder>hwMenu_newworldsel</zorder>
      <zorder>hwMenu_options</zorder>
      <zorder>hwMenu_optionssel</zorder>
      <zorder>hwMenu_quitsel</zorder>
      <zorder>hwMenu_quitmousearea</zorder>
      <zorder>hwMenu_optionsmousearea</zorder>
      <zorder>hwMenu_newworldmousearea</zorder>
      <zorder>hwMenu_loadworld</zorder>
      <zorder>hwMenu_loadworldsel</zorder>
      <zorder>hwMenu_loadworldmousearea</zorder>
      <zorder>hw_versionnumber</zorder>
     </widget>
     <widget class="QWidget" name="Loading">
      <widget class="QLabel" name="hw_menu_background_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1599</width>
         <height>900</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/HabitatWombat/Images/Menu/HabitatWombat/Images/Menu/hw_menu_background.jpg</pixmap>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>460</y>
         <width>200</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>430</y>
         <width>200</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Creating world...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="hw_versionnumber_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>870</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: orange;</string>
       </property>
       <property name="text">
        <string>v. 0.1.1 alpha</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
